src/index.html

```diff
@@ -6,9 +6,37 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin = "anonymous" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <header class="header">
+      <div class = "container" >
+        <a href="#home">
+          <img  class = "logo"
+                src="images/logo.png"
+               alt ="moyo-logo" >
+        </a>
+        <nav class = "nav">
+          <ul class ="nav__list">
+            <a class ="nav__link is-active" href ="index.html" >Apple</a>
+            <a class ="nav__link" href ="#" >Samsung</a>
+            <a class ="nav__link" href ="#" >Smartphones</a>
+            <a class ="nav__link"
+               href ="#"
+               data-qa="hover" >Laptops & Computers</a>
+            <a class ="nav__link " href ="#" >Gadgets</a>
+            <a class ="nav__link" href ="#" >Tablets</a>
+            <a class ="nav__link" href ="#" >Photo</a>
+            <a class ="nav__link" href ="#" >Video</a>
+          </ul>
+        </nav>
+      </div>
+    </header>
```

Comment:
This code must be written in the body tag

Reply:
thank you , I updated the code. I don't understand, why the ::after content is not being displayed after .is-active. 
Thank you